src/main/kotlin/org/jetbrains/intellij/platform/gradle/extensions/IntelliJPlatformDependenciesHelper.kt

```diff
@@ -793,6 +793,12 @@ class IntelliJPlatformDependenciesHelper(
         )
     }
 
+    // These two methods were created to make clearly visible that we want to use consistent version numbers in
+    // collectDependencies as well as in createIntelliJPlatformBundledModule & createIntelliJPlatformBundledPlugin
+    // When the version is retrieved like this, it is impossible to change it one place & forger to chang another.
+    private fun getVersionForBundledModule() = productInfo.get().buildNumber
+    private fun getVersionForBundledPlugin(plugin: IdePlugin) = requireNotNull(plugin.pluginVersion)
```

Comment:
This method seems redundant, as its usage always appears next to other statements like `requireNotNull(plugin.PROPERTY)`, so it's clear value should come from `plugin` object.